setup.py

```diff
@@ -39,7 +39,7 @@ def get_long_description():
     include_package_data=True,
     install_requires=[
         "anyio>=3.4.0,<4",
-        "typing_extensions; python_version < '3.10'",
+        "typing_extensions>=3.10.0; python_version < '3.10'",
```

Comment:
Using the style of the one below :sweat_smile: 
```suggestion
        "typing_extensions >= 3.10.0; python_version < '3.10'",
```

Reply:
Yeah I wondered about that, but the one above uses this style `anyio>=3.4.0` 😄

Reply:
Gonna just resolve this conversation given that we don't have a consistent style *either* way.
Would accept a PR bringing all the cases into line one way or the other. 🙃